pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -645,6 +650,45 @@ func parseDeviceAddress(addrString string) []string {
 	return addrs
 }
 
+func expandDiskImages(vmi *v1.VirtualMachineInstance, domain *api.Domain) {
+	logger := log.Log.Object(vmi)
+	for _, disk := range domain.Spec.Devices.Disks {
+		if imageCanBeExpanded(disk) {
+			logger.Infof("pre-start expansion of disk %v", disk)
```

Comment:
Probably a good idea to log the target size too.

Should the logging statement be moved to the function that actually performs the image resizing?